io/net/Network_config.py

```diff
@@ -47,14 +47,31 @@ def setUp(self):
             self.cancel("%s interface is not available" % self.iface)
         self.ipaddr = self.params.get("host_ip", default="")
         self.netmask = self.params.get("netmask", default="")
+        self.iface_type = self.params.get("iface_type", default="vio")
         configure_network.set_ip(self.ipaddr, self.netmask, self.iface)
         cmd = "ethtool -i %s" % self.iface
         for line in process.system_output(cmd, shell=True).decode("utf-8") \
                                                           .splitlines():
             if 'bus-info' in line:
                 self.businfo = line.split()[-1]
+            if 'driver' in line:
```

Comment:
@abdhaleegit Please use a different method, since this defeats the comparison done in `test_driver_check()`

Reply:
test_driver_check does a driver check between driver got from pci address and driver got from ethtool

any virtual network does not have a PCI address, so as the driver cannot be detected via pci address, I wanted to have driver taken from

yes, it destroys the need of driver check in case of virtual devices.  should I exclude driver check for virtual interface ?

Reply:
If there are 2 ways to check the driver in case of virtual interface, it would be good.
Else, try to cancel that test for virtual interfaces.